src/loki/worker.cc

```diff
@@ -84,7 +84,7 @@ void loki_worker_t::parse_costing(Api& api, bool allow_none) {
   } catch (const std::runtime_error&) { throw valhalla_exception_t{125, "'" + costing_str + "'"}; }
 
   // See if we have avoids and take care of them
-  if (options.avoid_locations_size() > max_avoid_locations) {
+  if ((size_t)options.avoid_locations_size() > max_avoid_locations) {
```

Comment:
Was this c-style cast suggested by clang? maybe do a `static_cast<size_t>` here?

Reply:
Oops, no suggestion, just an error about comparing different types. This is me being too used with C. I'll fix.

Reply:
:+1: